e2e/FR05-fr.e2e-spec.ts

```diff
@@ -2,7 +2,7 @@ import { by, browser, element } from 'protractor';
 
 import { Utils } from './utils/app.utils';
 
-describe('Zonemaster test FR06 - [Supports French language]', () => {
+describe('Zonemaster test FR06-fr - [Supports French language]', () => {
```

Comment:
I think this should be `FR05-fr` to be aligned with the rest.

Reply:
Corrected.